File: py/test2.py
import time
def factor(n):
    if n > 1:
        return n*factor(n - 1)
    else:
        return n
while 1:
    fc = 0
    ts = time.time()
    while time.time() - ts < 1:
        factor(100)
        fc = fc + 1
    print(fc)

File: js/test2.js
function factor(n) {
    return n > 1 ? n * factor(n - 1) : n
}
for (; ;) {
    var fc = 0, ts = (new Date).getTime()
    while ((new Date).getTime() - ts < 1000) {
        factor(100)
        ++fc
    }
    console.log(fc)
}